trieste/acquisition/function.py

```diff
@@ -528,15 +532,187 @@ def prepare_acquisition_function(
         return lambda at: expected_improvement(objective_model, eta, at) * constraint_fn(at)
 
 
-class BatchAcquisitionFunctionBuilder(ABC):
-    """ A :class:`BatchAcquisitionFunctionBuilder` builds a batch acquisition function. """
+class IndependentReparametrizationSampler:
+    r"""
+    This sampler employs the *reparameterization trick* to approximate samples from a
+    :class:`ProbabilisticModel`\ 's predictive distribution as
+
+    .. math:: x \mapsto \mu(x) + \epsilon \sigma(x)
```

Comment:
sigma(x) is misleading, it looks like the covariance matrix while it's the Cholesky decomposition. Use L(x) instead?

Reply:
are you sure? this is the independent case, and we're literally doing `tf.sqrt(cov)` with diagonal cov